github/prci.py

```diff
@@ -187,57 +84,123 @@ def load_yaml(yml_path):
     return parser
 
 
-def update_runner(repo, creds):
-    cmd = ['git', 'pull']
-    stdout = subprocess.check_output(cmd).decode('utf-8')
-    if 'Already up-to-date' not in stdout:
-        logger.info('Code change detected, re-deploying runner')
-        subprocess.call([
-            'ansible-playbook',
-            '-i', 'ansible/hosts/runner_localhost',
-            '-e', 'github_token={}'.format(creds['token']),
-            '-e', 'deploy_ssh_key=false',
-            'ansible/prepare_test_runners.yml'])
-
-
-def sentry_report_exception(context=None):
-    sentry = raven.Client(SENTRY_URL)
-
-    if context is not None:
-        with sentry.context:
-            sentry.context.merge(context)
-    try:
-        sentry.captureException()
-    finally:
-        sentry.context.clear()
-
-
-def check_reboot(repo, creds):
-    def plan_reboot(delay=REBOOT_DELAY):
-        next_reboot = int(time.time()) + delay
-        with open(REBOOT_TIME_FILE, 'w') as f:
-            f.write(str(next_reboot))
+def process_pull_request(pull_request, repository_url):
+
+    if pull_request.postponed:
+        skipping_pr("postponed.", pull_request.number)
+        return None
+
+    if not pull_request.mergeable:
+        skipping_pr("can't be merged.", pull_request.number)
+        if not pull_request.needs_rebase:
```

Comment:
Actually, is without the `not`, isn't it? If the PR cannot be merged and needs rebase, it adds the label.

Reply:
No, it's actually not https://github.com/freeipa/freeipa-pr-ci/pull/168/files#diff-d4101ce55c387d10a96f9a7088cbf8edR417